index.html

```diff
@@ -221,9 +235,21 @@ <h3 class="grouping-title">Attributes</h3>
               this value changes from its initially configured value, the camera
               will interpolate from its current value to the new value. Defaults
               to "auto", which sets either the vertical or horizontal field of
-              view to 45 degrees depending on the dimenions of the model and the
+              view to 45 degrees depending on the dimensions of the model and the
               aspect ratio of the canvas.</p>
             </li>
+            <li>
+              <div>field-of-view-min</div>
```

Comment:
This is very nit-picky, but let's do `min-*`/`max-*` rather than `*-min`/`*-max` to align with CSS-style e.g., `min-width`/`max-width`.